### ISSUE TITLE
Is this project still being maintained? 

### ISSUE THREAD

**Issue by ncsrikar** [fast-jsonapi/fast_jsonapi#250]
I am trying to find an alternative to AMS which for some reason no longer works in ruby 3.3.5. 
I don't see a push since 2023 and the v3 hasn't really been released.! 
Don't want to spend several days migrating only to find this not being supported by the time next big update comes along for ruby. 
Kindly clarify if this is being maintained in any capacity. 
Thank you!

**Comment by stas** [comment]
Yes, we're still using the project and have no issues (we're on ruby v3.3 and rails7). 

For more context see https://github.com/jsonapi-serializer/jsonapi-serializer/pull/141#issuecomment-1822616311

But please keep the conversation ontopic. So what's the issue you're having?

**Comment by night91** [comment]
Hopefully this get some maintenance some day. I can not find anything as complete and straightforward to use as this one. Other gems lack good features this one has.

Open to receive some recommendations of some one could find a replacement.

**Comment by night91** [comment]
Indeed, this is the bets project and there is not really alternative.

**Comment by stas** [comment]
fyi https://github.com/jsonapi-serializer/jsonapi-serializer/issues/254
